api/mapping/templates/mapping/scanreportvalue_list.html

```diff
@@ -2,64 +2,84 @@
 
 {% block body %}
 
-    <nav aria-label="breadcrumb">
-        <ol class="breadcrumb">
-            <li class="breadcrumb-item"><a href="{% url 'home' %}">Home</a></li>
-            <li class="breadcrumb-item"><a href="{% url 'scan-report-list' %}">Scan
+<nav aria-label="breadcrumb">
+    <ol class="breadcrumb">
+        <li class="breadcrumb-item"><a href="{% url 'home' %}">Home</a></li>
+        <li class="breadcrumb-item"><a href="{% url 'scan-report-list' %}">Scan
                 reports</a></li>
-            <li class="breadcrumb-item"><a href="{% url 'tables' %}?search={{ scan_report.id }}">
+        <li class="breadcrumb-item"><a href="{% url 'tables' %}?search={{ scan_report.id }}">
                 {{ scan_report.data_partner }} - {{ scan_report.dataset }}</a></li>
-            <li class="breadcrumb-item"><a href="{% url 'fields' %}?search={{ scan_report_table.id }}">
+        <li class="breadcrumb-item"><a href="{% url 'fields' %}?search={{ scan_report_table.id }}">
                 {{ scan_report_table.name }}</a></li>
-            <li class="breadcrumb-item"><a href="{% url 'values' %}?search={{ scan_report_field.id }}">
+        <li class="breadcrumb-item"><a href="{% url 'values' %}?search={{ scan_report_field.id }}">
                 {{ scan_report_field.name }}</a></li>
 
-        </ol>
-    </nav>
+    </ol>
+</nav>
 
-    <h2>Values</h2>
-
-    <div>
-        <table class="table" style="float:left;">
-            <thead>
+<h2>Values</h2>
+{{scan_report_concept}}
```

Comment:
Do we need it here?

Reply:
Nope, deleted! Must've been a hangover from something I was testing.